core/drivers/fabric-driver/server/fabric-code.ts

```diff
@@ -77,6 +77,8 @@ const getNetworkGateway = async (networkName: string): Promise<Gateway> => {
 async function invoke(
     query: query_pb.Query,
     networkName: string,
+    funName: string,
+    dynamicArg: Buffer | undefined
```

Comment:
I think to make it optional, we can use `?` here.
Otherwise we still will need to pass `undefined` here. If we use `?`, then by default it will be undefined.

Reply:
Done Sandeep.